support-frontend/assets/pages/paper-subscription-checkout/components/addDigiSubCta.tsx

```diff
@@ -141,6 +150,14 @@ function AddDigiSubCta({ addDigitalSubscription, digiSubPrice }: PropTypes) {
 					/>
 				</div>
 			</AccordionRow>
+
+			{/* The Accordian expects multiple children so this hack will appease the ts compiler for now */}
+			<AccordionRow
+				label={''}
+				cssOverrides={css`
+					display: none;
+				`}
+			/>
```

Comment:
Maybe we could wrap our original row in a list? i.e

```tsx
{[
  <AccordionRow />
]}
```

Reply:
I'm not too sure of the benefit of doing that?

Reply:
It would remove the need for the second `AccordionRow`

Reply:
Oh right, I suppose that's down to the underlying source component implementation?

Reply:
Yeah so the `Accordion` component in source wants a list of children. So we can either give it two children like we've got here, or give it a list containing just a single child. I think it might be slightly nicer to just give it a list to avoid the `display: none` component being added to the dom